pytensor/link/numba/dispatch/elemwise.py

```diff
@@ -428,10 +434,17 @@ def dimshuffle(x):
                     new_strides = numba_basic.tuple_setitem(
                         new_strides, i, old_strides[o]
                     )
+            if drop:
+                for dropped_dim in drop:
+                    if old_shape[dropped_dim] != 1:
+                        raise ValueError(
+                            "DimShuffle: Attempting to squeeze axes with size not equal to one"
```

Comment:
Does this raise in other backends? I thought it's meant to just do nothing

Reply:
It does, you're thinking of `pt.squeeze(axis=None)` not `pt.squeeze(axis=something)`. If you say something it will try to squeeze, you basically commited to a shape of 1 there even if not known